src/cli.ts

```diff
@@ -2,38 +2,59 @@
  * @copyright Microsoft Corporation. All rights reserved.
  */
 import yargs from 'yargs/yargs';
+import { validate as uuidValidate } from 'uuid';
 import { run } from '.';
 
-interface Arguments {
+interface IArguments {
   ID: string;
+  beforeDate: string;
   projectPath: string;
   ksFilePath: string;
 }
 
-const argv = yargs(process.argv.slice(2))
-  .usage('ks-killer <KS ID> [options...]')
-  .showHelpOnFail(true, 'Please provide a KS ID')
-  .command(
-    '$0 <ID>',
-    'KS Killer',
-    (yargs) => {
-      yargs
-        .positional('id', {
-          describe: 'ID of the KS to graduate',
-          type: 'string'
-        })
-        .option('project-path', {
-          alias: 'p',
-          type: 'string',
-          describe: 'Target project to clean. Default to cwd.',
-          default: process.cwd()
-        })
-        .option('ks-file-path', {
-          alias: 'k',
-          type: 'string',
-          describe: 'The file path of the KS. This boosts performance.'
-        });
-    },
-    ({ ID, projectPath, ksFilePath }: Arguments) => run(ID, projectPath, ksFilePath)
-  )
+const argv: IArguments = yargs(process.argv.slice(2))
+  .usage('ks-killer [options...]')
+  .showHelpOnFail(true)
+  .option('ID', {
```

Comment:
Now all of the are `options`, which doesn't cover the use case `ks-killer <ks-id>`. We need to update the usgae.